l10n_th_personal_income_tax/models/pit_move.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2021 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+from odoo.addons.l10n_th_withholding_tax_cert.models.withholding_tax_cert import (
+    WHT_CERT_INCOME_TYPE,
+)
+
+
+class PersonalIncomeTaxMove(models.Model):
+    _name = "pit.move"
+    _description = "Personal Income Tax Move"
+
+    payment_id = fields.Many2one(
+        comodel_name="account.payment",
+        string="Payment",
+        index=True,
+        required=True,
+        ondelete="cascade",
+        domain=[("state", "not in", ["draft", "cancel"])],
```

Comment:
This model has `state` field? I don't see.

Reply:
state on model `account.payment`